includes/subscriptions.php

```diff
@@ -280,7 +280,16 @@ function send_notifications( $post_id ) {
 		$request = wp_remote_post(
 			untrailingslashit( $target_url ) . '/wp/v2/dt_subscription/receive',
 			[
-				'timeout' => 5,
+				/**
+				 * Filter the timeout used when calling `\Distributor\Subscriptions\send_notifications`
+				 *
+				 * @hook dt_subscription_post_timeout
+				 *
+				 * @param {int} $timeout The timeout to use for the remote post. Default `5`.
+				 *
+				 * @return {int} The timeout to use for the remote post.
+				 */
+				'timeout' => apply_filters( 'dt_subscription_post_timeout', 5 ),
```

Comment:
What about adding `$post` as an argument of this filter, this may help some edge cases (I haven't think about) and make this filter align with `dt_push_post_timeout` filter.

Reply:
@dinhtungdu thanks for review. Added.